components/policies/PolicyForm.js

```diff
@@ -119,7 +119,7 @@ const PolicyForm = ({
     const policy = await response.json();
 
     dispatch({
```

Comment:
Not sure why exactly, but it seems like the policy version history isn't updating after a new version is made.

Reply:
Hmmmm.... I am not able to reproduce. Could you show me?

Reply:
Never mind me, I could only get it to happen with a single policy and can't reproduce it after nuking `.next`.